### PR TITLE
expanded projects page to encourage in-progress additions

### PR DESCRIPTION
**Pull request by Phildo**
feel free to remove my entry (gnome tower), as it's very early in development and lacks relative depth of available content. I've only included it to encourage others of similarly incomplete status to add their own content to the list.

### CODE REVIEW

**Comment by Longor1996** [comment]
Please fixup your changes to align with the changes from 446b17d52843b20d7063255204f92e6fdb335213

**Comment by Phildo** [comment]
updated to new format, added discord submissions

File: content/wiki/projects.toml [hunk 1]
```diff
@@ -75,3 +75,52 @@ language = ["Javascript"]
...
+state = "Active Dev."
+language = ["D", "Vox using Vulkan"]
+source = "https://github.com/MrSmith33/voxelman2"
+license = "Boost"
+
+[[projects]]
+name = "Dream Settler"
+url = "dreamsettler.com"
+state = "Active Dev."
+language = ["Unreal"]
```
Comment (Longor1996) [inline content/wiki/projects.toml]: Shouldn't this be `engine`?

File: content/wiki/projects.toml [hunk 2]
```diff
@@ -75,3 +75,52 @@ language = ["Javascript"]
...
+state = "Active Dev."
+language = ["C++"]
+source = "https://mgerhardy.github.io/vengi"
+license = "MIT"
+
+[[projects]]
+name = "Colony Survival"
+url = "https://store.steampowered.com/app/366090/Colony_Survival/"
+state = "Early Access"
+language = ["Unity3d", "C#"]
```
Comment (Longor1996) [inline content/wiki/projects.toml]: Unity 3D is an engine, not a language.

File: content/wiki/projects.toml [hunk 3]
```diff
@@ -75,3 +75,52 @@ language = ["Javascript"]
...
+url = "https://www.youtube.com/channel/UC6qe1IBLMBi0jqaec-Msxlw"
+state = "Active Dev."
+language = ["C++"]
+rendering = "Rasterization"
+
+[[projects]]
+name = "Voxelman"
+url = "https://github.com/MrSmith33/voxelman2"
+state = "Active Dev."
+language = ["D", "Vox using Vulkan"]
```
Comment (Longor1996) [inline content/wiki/projects.toml]: "using Vulkan" is not part of the language, but rather the graphics API/rendering.

**Review by Longor1996 — CHANGES_REQUESTED** [review]
Plz fix

**Comment by Phildo** [comment]
resolved

**Comment by Longor1996** [comment]
Looks good! Merging...

**Review by Longor1996 — APPROVED** [review]

**Pull request merged**